src/EOS.hpp

```diff
@@ -8,23 +8,29 @@
 /// \brief A class for equation of state calculations.
 
 #include <cmath>
+#include <string>
 
 #include "AMReX_GpuQualifiers.H"
 #include "AMReX_REAL.H"
+#include "burn_type.H"
+#include "eos.H"
+#include "extern_parameters.H"
 
 namespace quokka
 {
 static constexpr double boltzmann_constant_cgs = 1.380658e-16; // cgs
 static constexpr double hydrogen_mass_cgs = 1.6726231e-24;     // cgs
+static constexpr double cs_isothermal = NAN;		       // only used when gamma = 1
+static constexpr double mean_molecular_weight = NAN;
```

Comment:
I don't know what to do about `cs_isothermal` and `mean_molecular_weight`, so I moved them from EOS_Traits to the namespace itself.

Reply:
`mean_molecular_weight` is only used to compute the temperature, so we should replace wherever it's used with calls to the EOS temperature function.

`cs_isothermal` is only used by the Riemann solver for tests that assume an isothermal equation of state. I'm not sure of the best way to replace this in the code.